packages/klevu-ui/src/components/klevu-facet/klevu-facet.tsx

```diff
@@ -15,6 +15,21 @@ import {
 import { getGlobalSettings } from "../../utils/utils"
 
 export type KlevuFacetMode = "checkbox" | "radio"
+export type OPTION_TYPE = {
```

Comment:
This should be in `PascalCase` and start with `Klevu` because exported types will be visible for library users.

But actually this type is not used in any other place so we can drop `Klevu` and remove `export` from the beginning.

```suggestion
type OptionType = {
```

Reply:
changed